db/init_db.js

```diff
@@ -1,31 +1,193 @@
 // code to build and initialize DB goes here
 const {
-  client
+  client,
   // other db methods 
 } = require('./index');
 
-async function buildTables() {
+const {createUser} = require('./db')
+const {createProduct} = require('./products')
+
+async function dropTables() {
+  client.connect();
+  console.log("Dropping Tables ...")
   try {
-    client.connect();
+    await client.query(`
+      DROP TABLE IF EXISTS order_products;
+      DROP TABLE IF EXISTS orders;
+      DROP TABLE IF EXISTS products;
+      DROP TABLE IF EXISTS users;
+    `);
+  } catch (error) {
+    throw error
+  }
+}
 
-    // drop tables in correct order
+async function buildTables() {
+  console.log("Building Tables ...")
+  try {
+    await client.query(`
+      CREATE TABLE users(
+        id SERIAL PRIMARY KEY,
+        "firstName" VARCHAR(255) NOT NULL,
+        "lastName" VARCHAR(255) NOT NULL,
+        email VARCHAR(255) UNIQUE NOT NULL,
+        imageURL VARCHAR(255) DEFAULT 'imageUrl',
+        username VARCHAR(255) UNIQUE NOT NULL,
+        password VARCHAR(255) UNIQUE NOT NULL,
```

Comment:
I don't think passwords need to be unique.

Reply:
This was the line from the task list for table definitions i was referring to: "Create a users table definition with the following information: password - unique; not null". 
But I guess either way it doesn't really matter. Unique for password would just be preventing duplicate passwords, right? The chances of someone creating the same password as another user would be slim so do we want to prevent that from happening? If so, I'll leave it but if you feel it's not necessary I can take it out!

Reply:
I agree, not likely to cause any issues. We can leave it, then. Thanks!